test/python/plugins/qiskit/test_mock_backend.py

```
@@ -267,7 +267,7 @@ def supported_program_formats() -> list[ProgramFormat]:
         """Return list of supported program formats."""
         return [ProgramFormat.QASM2, ProgramFormat.QASM3]
 
-    def submit_job(self, program: str, program_format: ProgramFormat, num_shots: int) -> MockJob:  # ruff:ignore[unused-method-argument]
+    def submit_job(self, program: str, program_format: ProgramFormat, num_shots: int) -> MockJob:
```

Ruff (0.16.1)

[warning] Unused method argument: `program_format`

(ARG002)